dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/PLL/PLL.mo

```diff
@@ -0,0 +1,125 @@
+within Dynawo.Electrical.Controls.PLL;
+
+model PLL "Phase-Locked Loop"
+
+  import Modelica.Blocks;
+  import Modelica.ComplexBlocks;
+  import Modelica.Constants;
+  import Modelica.SIunits;  
+    
+  import Dynawo.Connectors;
+
+  parameter SIunits.PerUnit Kp "PLL voltage calculation proportional gain";
+  parameter SIunits.PerUnit Ki "PLL voltage calculation integrator";
+  parameter SIunits.PerUnit fmin "Lower frequency limit (only positive values!)";
```

Comment:
in our naming convention, we have f in Hz (around 50) , omega in rad/s (around 2Pi*50) and omegaPu in pu (around 1), maybe fmin and fmax should be renamed as OmegaMaxPu and OmegaMinPu (starting with capital letter because they are parameters)

Reply:
ok, done